talk/morelanguage/constexpr.tex

```diff
@@ -14,8 +14,10 @@
         if (x > 1) return x * f(x - 1);
         return 1;
       }
-      constexpr int a = f(5); // computed at compile time
-      int b = f(n); // computed at runtime
+      constexpr int a = f(5); // computed at compile-time
+      int b = f(5); // maybe computed at compile-time
+      int n = ...;  // runtime value
+      int c = f(n); // computed at runtime
```

Comment:
This might not fit on the slide, but I like having it here, so we should look how to shrink other things a bit.